src/game_client.cpp

```diff
@@ -60,8 +60,8 @@ namespace ace {
             SDL_ERROR("IMG_Init");
 
         SDL_GL_LoadLibrary(nullptr);
-//        SDL_GL_SetAttribute(SDL_GL_CONTEXT_MAJOR_VERSION, 3);
-//        SDL_GL_SetAttribute(SDL_GL_CONTEXT_MINOR_VERSION, 3);
+        SDL_GL_SetAttribute(SDL_GL_CONTEXT_MAJOR_VERSION, 3);
```

Comment:
On my machine, leaving this out loads the latest version of OpenGL available. I like this because it allows me to use higher version OpenGL features conditionally without breaking it for other systems (e.g. the debug callback stuff later down this function). Is there any other possible workaround for this? If not, I think it would be best to keep it behind an ifdef.

Reply:
The Mesa drivers on Linux have microsoft's hands in their pockets, so you have to request any version above something like 2.1 if you're not using an intel gpu. Once I uncommented this, the version it seems to actually give is 4.1